### DISCUSSION TITLE
Table Metrics to have pagination and show link if single action

### DISCUSSION THREAD

**Discussion by ziming** [laravel/nova-issues#4610] [Ideas & Feature Requests]
Suggestion: Table Metrics to have pagination and show link if single action instead of 3 vertical dots

If will be really nice if it can be like filament where we can embed a real dynamic table with columns on the dashboard too.

**Comment by kusab85** [comment]
FR bump!

**Comment by ao-jhelmich** [comment]
Great QOL improvement, if only 1 link is shown show it directly instead of first clicking the ...
